database/setup.sh

```
@@ -1,12 +1,12 @@
-#
-#   EXECUTE
-#
-
-# EXECUTE -> RESET DATABASE
-mysql -u root -p < /srv/www/website/database/reset.sql
-
-# EXECUTE -> INJECT DATABASE
-mysql -u root -p abscissa < /srv/www/website/database/database.sql
-
-# EXECUTE -> DATABASE INITIAL STATE
+#
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)